sealed_sector_health/health.go

```diff
@@ -0,0 +1,25 @@
+package sealed_sector_health
+
+// Health represents the healthiness of a sector managed by a
+// sector builder.
+type Health int
+
+const (
+	Unknown              Health = iota
+	Ok                          // everything is fine
+	ErrorInvalidChecksum        // sector exists, but checksum is invalid
```

Comment:
Since `Ok` is really the only valid state, it's probably fine to drop the `Error` prefix

Reply:
Sounds good.